dev-python/lazy-object-proxy/lazy_object_proxy-1.5.2.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A fast and thorough lazy object proxy for Python"
+DESCRIPTION="lazy-object-proxy is based on the ObjectProxy from the wrapt \
+Python module with one big change: it calls a function the first time the \
+proxy object is used, while wrapt.ObjectProxy just forwards the method calls \
+to the target object"
+HOMEPAGE="https://github.com/ionelmc/python-lazy-object-proxy"
+COPYRIGHT="2014-2019, Ionel Cristian Mărieș"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/ionelmc/python-lazy-object-proxy/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="b28708f92930b4e2c0ab39def12ebc6be466db5e91fad88bfc7ebf9aa9c9de16"
+SOURCE_DIR="python-lazy-object-proxy-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+
+PYTHON_PACKAGES=(python python36 python3)
+PYTHON_VERSIONS=(2.7 3.6 3.7)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+	pythonPackage=${PYTHON_PACKAGES[i]}
+	pythonVersion=${PYTHON_VERSIONS[$i]}
+
+	eval "PROVIDES_$pythonPackage=\"
+		${portName}_$pythonPackage = $portVersion
+		\""
+	eval "REQUIRES_$pythonPackage=\"
+		haiku
+		cmd:python$pythonVersion\
+		\""
+	BUILD_REQUIRES="$BUILD_REQUIRES
+		setuptools_$pythonPackage
+		cmd:git"
```

Comment:
cmd:git shouldn't be required.

Reply:
Hmm, yeah I was thinking the same thing. I tried removing it again and read through the error message of the failed build a bit more carefully and found it was asking for `setuptools_scm*`. Is this fix correct? I've pushed this onto the branch. (Had to disable python2.7 though as it lacks a `setuptools_scm`).

Thanks!